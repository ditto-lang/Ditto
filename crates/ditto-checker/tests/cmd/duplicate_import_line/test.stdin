module Test exports (..);

import Dep;
import Dep;

dep_five = Dep.five;
