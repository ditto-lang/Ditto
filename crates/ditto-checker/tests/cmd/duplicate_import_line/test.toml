bin.name = "ditto-checker-testbin-check-module"
args = ["duplicate_import_line.ditto"]
fs.sandbox = true
status = "failed"
