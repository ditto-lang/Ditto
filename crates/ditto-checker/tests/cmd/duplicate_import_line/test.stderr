
  × duplicate import
   ╭─[duplicate_import_line.ditto:2:1]
 2 │ 
 3 │ import Dep;
   · ─────┬────
   ·      ╰── previous import
 4 │ import Dep;
   · ─────┬────
   ·      ╰── duplicated here
 5 │ 
   ╰────

